backend/src/libs/interfaces/fastify-request.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { type FastifyRequest } from 'fastify';
+
+interface IFastifyRequest extends FastifyRequest {
+  user?: {
+    id: number;
+    email: string;
+  } | null;
+}
+
+export { type IFastifyRequest };
```

Comment:
Instead of type assertion - determine type in `index.d.ts` file - this file should be added in `backend/src/`
it may look like this:
```
import { type UserAuthResponse } from './packages/users/users.js';

declare module 'fastify' {
  interface FastifyRequest {
    user: UserAuthResponse;
  }
}
```
```suggestion

```